app/models/product.rb

```diff
@@ -498,6 +498,7 @@ def set_initial_values
   # Try to find the best name for the new products
   def choose_default_name
     return if name.present?
+    return self.name = "blank" if name.blank?
```

Comment:
Why don't you just leave the name blank?

Reply:
Hello,
I have try to leave blank name, But name validation presence: true, So  we are not able to create Product , And Here I am not find interventions and its details so unable to set the name value that the reason I will pass "blank" text and On the InterventionOutput I get this name and update its value accordingly,